IFC4x3/Constants/v/VIENNESEBEND_1wHLHEhpb2GhA0VAasbHUK.xml

```diff
@@ -1,3 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="VIENNESEBEND_1wHLHEhpb2GhA0VAasbHUK" Name="VIENNESEBEND" UniqueId="7a45544e-af39-4242-b280-7ca936951794" />
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="VIENNESEBEND_1wHLHEhpb2GhA0VAasbHUK" Name="VIENNESEBEND" UniqueId="7a45544e-af39-4242-b280-7ca936951794">
+	<Documentation>The Viennese Bend (R) is an innovative track geometry transition element. Instead of analyzing the vehicle movement at the track plane the optimization efforts target a gravity center line at a defined height above the rails.
+
+As a consequence the path of the horizontal alignment center line is also influenced by the cant layout. The first part of the curvature formula is assembled from the basic function like the other transition bends. The additional term contains the bank angle &quot;&amp;psi;&quot; and the gravity center line height &quot;h&quot; and is unique to the Viennes Bend (R). This term causes a small movement contrary to the main direction in the x,y layout. 
```

Comment:
height=>elevation?

Reply:
Hmmm ... in this case the original German term is "Trassierungshöhe". It denotes a local geometry property placing the line central to kinematic analysis relative to track plane. I am not a native speaker but discussion  in https://en.wikipedia.org/wiki/Height makes me think that both "elevation" and "altitude" have a slightly different meaning.

Reply:
I think in this context 'height' is the correct term.